web/themes/joinup/prototype/sass/components/_fieldset.sass

```diff
@@ -55,11 +55,12 @@
 
 .fieldset--border-bottom
   position: relative
-  padding-bottom: 18px
+  padding-bottom: 27px
```

Comment:
I'd recommend you separate spacing out of a selector named "border-bottom".
It can be better adding a modifier responsible for spacing.
Some other modifiers such as the `fieldset--box` also have spacing which can be delegated to the same modifier.
In overall, the rules about padding/margin in this component don't look very consistent. This is not a stopper, but if I had to maintain this I would be very afraid to change anything in these rules not being able to follow what's really going on.